src/test/java/uk/gov/service/notify/ClientIntegrationTestIT.java

```diff
@@ -19,39 +18,41 @@
     @Test
     public void testEmailNotificationIT() throws NotificationClientException, InterruptedException {
         NotificationClient client = getClient();
-        NotificationResponse emailResponse = sendEmail(client);
-        Notification notification = client.getNotificationById(emailResponse.getNotificationId());
+        SendEmailResponse emailResponse = sendEmailAndAssertResponse(client);
+        Notification notification = client.getNotificationById(emailResponse.getNotificationId().toString());
         assertNotification(notification);
```

Comment:
should these be using the new `assertNotificationEmailResponse` and `assertNotificationSmsResponse` functions? and should those functions, in turn, be more complete, perhaps calling through to `assertNotification` to check the template, notification_id, etc fields?